client/src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import React from 'react';
+import { Input, Button } from 'reactstrap';
+import './LoginForm.css';
+import { Field, reduxForm } from 'redux-form';
+import renderField from '../RenderField';
+import validate from './LoginValidator';
+
+type Props = {
+  onSubmit(e: any): void,
+};
+
+let LoginForm = (props: Props) => {
+  const { valid, onSubmit } = props;
+  console.log(props);
+  return (
+    <form id="login-form" onSubmit={onSubmit}>
```

Comment:
why do we need the id?

Reply:
for css, to limit the styles in scope of this particular id, otherwise it scatters around the app.

Reply:
arnt styles encapsulated to the component?